source/features/clean-rich-text-editor.tsx

```diff
@@ -2,5 +2,25 @@ import './clean-rich-text-editor.css';
 import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager.js';
+import select from 'select-dom';
+
+function hideToolTip(): void {
+    for (const textarea of select.all(`
+        input[id="issue_title"], 
```

Comment:
The selector for id starts with #

Please replace all of these to look like `#issue_title` or `input#issue_title` (I don’t think the tag name is necessary)

Also select.all() accepts an array of strings. You can use that to improve wrapping and allow comments. You can refer to the other features to see how it can be done